src/server/tiering/external_alloc.cc

```diff
@@ -284,7 +292,7 @@ class ExternalAllocator::SegmentDescr {
     auto FindPageSegment() -> Page* {
       for (uint32_t i = 0; i < capacity; ++i) {
         if (!pages[i].segment_inuse) {
-          pages[i].segment_inuse = 1;
+          pages[i].segment_inuse = true;
```

Comment:
looks that you changed deliberately, why?

Reply:
because why use bitfields if you only use it as a boolean, changed it back 🙄